app/src/components/LoginRegister/choose.jsx

```diff
@@ -1,13 +1,27 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import "./loginregister.css";
+import { Auth0Client } from "@auth0/auth0-spa-js";
 
 const Choose = () => {
+  const auth0Login = async (event) => {
+    event.preventDefault();
+
+    const configResponse = await fetch("/api/oauth/config");
```

Comment:
Given that we have a few methods in the SDK client which don't need authorization (like register), shouldn't this fetch also be done via a method in the SDK rather than directly here from the component?